pymtl3/passes/sverilog/import_/ImportPass.py

```diff
@@ -206,12 +206,15 @@ def create_verilator_c_wrapper( s, m, full_name, packed_ports, dump_vcd, cached
     port_inits = '\n'.join( port_inits )
 
     # Fill in the C wrapper template
```

Comment:
It might be better to write two paragraphs here. One is what you are doing. One is what we are planning to do about the 2-level caching scheme